client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,7 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
+	<span class="from"><span class="expander"></span></span>
```

Comment:
>  With #1047 you absolutely can not click on the text (or even on the empty space left) because

I have seen you point out multiple times issues that are only related to that specific PR, and not this one. While the idea of that PR is great, it's really some subpar code at the moment. So to get that one merged, this one has to be merged first, and work needs to be done on the other one.

@astorije's changes do make sense, and they simplify things, and they absolutely do not make it harder for #1047 to work. It should be just as simple to move the arrow to beginning of the line, instead of the end, to fix the jumping button problem. I do think that having clickable arrow in nickname row as it was before is not a very good solution, as it may be confusing and not that obvious that it does something unique compared to other icons there.